kong/plugins/aws-lambda/schema.lua

```diff
@@ -96,7 +96,6 @@ return {
   } },
   entity_checks = {
     { mutually_required = { "config.aws_key", "config.aws_secret" } },
-    { mutually_exclusive = { "config.aws_region", "config.host" } },
```

Comment:
By allowing both fields to be present will break version compatibility with older data planes. Special care will need to be made when determining which of these fields should be removed for older data planes when compatibility filters are added.

Reply:
The way I define a backwards-breaking change is: the customer upgrades kong, and *does nothing else*, and things don't work as before the upgrade (a blurry definition; not the important part of this argument).

If they have an old system and they upgrade, by definition, they will be using *either* the region or the host. So if they upgrade and *do nothing else* their old data planes should not need to be protected.

On the other hand, if they upgrade *and at the same time they change their plugin configuration to have both a host and a region*, then they will receive an error on their old Dataplanes. But at that point that is not a backwards-compatibility problem, since they *changed something else* besides doing the upgrade.